Example/SmileID/OnboardingScreen.swift

```diff
@@ -5,118 +5,148 @@ struct OnboardingScreen: View {
                 }
-                if #available(iOS 16.0, *) {
-                    content
-                        .presentationDetents([.medium])
-                        .presentationDragIndicator(.visible)
-                } else {
-                    content
-                }
-            }
-            .sheet(isPresented: $showQrCodeScanner) {
-                CodeScannerView(
-                    codeTypes: [.qr],
-                    scanInterval: 1,
-                    showViewfinder: true
-                ) { response in
-                    if case let .success(result) = response {
-                        let configJson = result.string
-                        if updateSmileConfig(configJson) {
-                            showQrCodeScanner = false
+                .sheet(isPresented: $showQrCodeScanner) {
+                    CodeScannerView(
+                            codeTypes: [.qr],
+                            scanInterval: 1,
+                            showViewfinder: true
+                    ) { response in
+                        if case let .success(result) = response {
+                            let configJson = result.string
+                            let updateResult = updateSmileConfig(configJson)
+                            partnerId = updateResult.1
+                            if updateResult.0 {
+                                showSuccess = true
+                                showQrCodeScanner = false
+                            }
                         }
                     }
                 }
-            }
+                .overlay(
+                        Group {
+                            if showSuccess {
+                                Color.black.opacity(0.3)
+                                        .edgesIgnoringSafeArea(.all)
+                                        .overlay(
+                                                AlertView(
+                                                        icon: Image(systemName: "checkmark.circle.fill"),
+                                                        title: "Configuration Added",
+                                                        description: "Welcome partner \(partnerId), you can now proceed to the home screen of the Sample App",
+                                                        buttonTitle: "Continue",
+                                                        onClick: {
+                                                            showSuccess = false
+                                                        }
+                                                )
+                                                        .padding([.leading, .trailing], 20)
+                                        )
+                            }
+                        }
+                )
     }
 }
 
-private func updateSmileConfig(_ configJson: String) -> Bool {
+private func updateSmileConfig(_ configJson: String) -> (Bool, String) {
```

Comment:
This is redundant, we don't need to return a tuple here. We can instead just return `String?`. If not nil, it means the config was successfully read. Otherwise if it is nil, that inherently means there was an error

Reply:
Fixed